test/dialect/postgresql/test_reflection.py

```diff
@@ -932,6 +932,49 @@ def go():
             ],
         )
 
+    @testing.provide_metadata
+    def test_index_reflection_partial(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

use connection fixture:

    def test_index_reflection_partial(self, connection):

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142

Reply:
**Ramon Williams** (RamonWill) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142